AnaClos.HabitTracker/AnaClos.HabitTracker/Program.cs

```diff
@@ -0,0 +1,337 @@
+            Console.ReadLine();
+            break;
+        case "d":
+            Delete(connectionString);
+            break;
+        case "u":
+            Update(connectionString);
+            break;
+        case "q":
+            Console.WriteLine("By!!!");
+            break;
+    }
+    choice = Menu();
+}
+
+void CreateBaseAndTable(string connectionString)
+{
+    using (var connection = new SqliteConnection(connectionString))
+    {
+
+        connection.Open();
+        var tableCmd = connection.CreateCommand();
+
+        tableCmd.CommandText = 
+            @"CREATE TABLE IF NOT EXISTS drinking_water (
+            Id INTEGER PRIMARY KEY AUTOINCREMENT, 
+            Date TEXT,
+            Quantity INTEGER
+            )";
+
+        tableCmd.ExecuteNonQuery();
+        connection.Close();
+    }
+}
+
+string Menu()
+{
+    string choice = string.Empty;
+
+    Console.Clear();
+    Console.WriteLine("Select your choice\n");
+    Console.WriteLine("(I) Insert Habit");
+    Console.WriteLine("(D) Delete Habit");
+    Console.WriteLine("(U) Update Habit");
+    Console.WriteLine("(V) View All Habits");
+    Console.WriteLine("(Q) Quit\n");
+    Console.WriteLine("------------------------------------------\n");
+
+    choice = Console.ReadLine().ToLower().Trim();
+
+    return choice;
+}
+
+void View(string connectionString)
+{
+    using (var connection = new SqliteConnection(connectionString))
+    {
+        List<Habit> list = new List<Habit>();
+
+        connection.Open();
+        var tableCmd = connection.CreateCommand();
+
+        tableCmd.CommandText =
+            "SELECT * FROM drinking_water";
+
+        SqliteDataReader reader = tableCmd.ExecuteReader();
+        if (reader.HasRows)
+        {
+            while(reader.Read())
+            {
+                list.Add(new Habit { Id = reader.GetInt32(0),Date=reader.GetString(1),Quantity=reader.GetInt32(2)});
+            }
+        }        
+
+        connection.Close();
+        Console.WriteLine();
+        foreach (var item in list)
+        {
+            Console.WriteLine(item.ToString());
+        }        
+    }
+}
+
+void Insert(string connectionString)
+{
+    string date = string.Empty;
+    string response = string.Empty;
+    int quantity = 0;
+    bool ok = false;
+    while( !ok )
+    {
+        date = GetDateInput();
+        ok = ValidateDate(date);
+    }
+    if (date == "r")
+    {
+        return;
+    }
+    ok = false;
+    while( !ok ) 
+    {
+        response = GetInput("Enter the Quantity of glasses you drink");
+        ok = ValidateInt(response);
+    }
+    if (response == "r")
+    {
+        return;
+    }
+    quantity = Int32.Parse(response);
+
+    var myHabit = new Habit { Date = date, Quantity = quantity };
+
+    using (var connection = new SqliteConnection(connectionString))
+    {
+
+        connection.Open();
+        var tableCmd = connection.CreateCommand();
+
+        tableCmd.CommandText =
```

Comment:
This code is open to [SQL Injection Attacks](https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html)

Remember to parameterise your queries:
```
// Open to SQL Injection Attacks.
command.CommandText = $"INSERT INTO Table(Column) VALUES ('{value}');";

// vs

// Closed to SQL Injection Attacks.
command.CommandText = $"INSERT INTO Table(Column) VALUES (@Value);";
command.Parameters.AddWithValue("@Value", value);
```